xml/2018-10-01.xml

```diff
@@ -0,0 +1,232 @@
+    weekly digest of the year contains a whole lot of useful information, so
+    keep on reading!
+  </text>
+
+  <hr/>
+
+  <heading>What is DoCSoc?</heading>
+
+  <text>
+    **Returning student?** If you aren't a fresher, feel free to skim through this
+    section. If you're new to DoCSoc, there's lots of useful information here so
+    read on!
+
+    **DoCSoc is the Department of Computing Society**. We run multiple events through
+    the year with our sponsors. We invite notable tech speakers, connect our
+    students with different opportunities and run a number of social events to
+    bring together the community of students who are interested in Computing.
+
+    **Committee -** This year, DoCSoc is brought to you by:
+
+    - **President**: Inara Ramji
+    - **Vice President**: Alessandro Bonardi
+    - **Treasurer**: Rachel Mekhteiva Lee
+    - **Industrial Events Director**: Rohan Pritchard
+    - **Social Events Director**: Jayati Sarkar
+    - **Academic Events Director**: Jack Pordi
+    - **Secretary**: Kelvin Zhang
+    - **Events Officers**: Szilvester Szekely, Serene Chongtrakul, Anindita (Andy) Ghosh
+    - **Marketing &amp; Publicity Officer**: Fawaz Shah
+    - **Webmaster**: George Soteriou
+    - **GameDev Officer**: Athi Dharma
+
+    **Dep Reps -** In addition to the DoCSoc committee, we have three departmental representatives
+    (dep reps) this year responsible for representing the student body to the department:
+
+    - **Computing**: Martin Zlocha ([email](mailto:dep-rep@doc.ic.ac.uk))
+    - **Joint Maths and Computing (JMC)**: Rachel Mekhtieva Lee and Fawaz Shah ([email](mailto:dep-rep@doc.ic.ac.uk))
+    - **Electronic and Information Engineering (EIE)**: Willem van der Schoot (Academic, [email](willem.van-der-schoot16@imperial.ac.uk)) and Mayank Surana (Wellbeing, [email](mayank.surana16@imperial.ac.uk))
+
+    If you have any academic or wellbeing related queries, your dep reps are people
+    you can talk to.
+
+    **Events -** DoCSoc runs a huge number of events every year. Our main events
+    include:
+
+    - **Talks and Workshops** - Not to be confused with the department's lunchtime
+    ACI talks, DoCSoc runs engaging talks and workshops with our sponsors, usually
+    in the evening with pizza (with dietary requirements catered for) and soft
+    drinks provided. In the past, we've run workshops such as capture the flag competitions
+    with Netcraft, an _Extreme Startup_ event with Palantir and Bloomberg's _CodeCon_.
+
+    - **Bar Nights** - Our most well-known events include our bar nights which
+    usually occur every month during term time. Bar nights are a chance for you to
+    socialise with your coursemates, make new friends and occasionally network with
+    recruiters sponsoring our bar nights. Bar nights are _free entry_ to our members
+    and you will be given tokens for _free_ drinks (both alcoholic and non-alcoholic)
+    to use at the bar!
+
```

Comment:
- **Sponsors' Exhibition ** - This is DoCSoc's Careers Fair, where our sponsors will come together to present their companies and interact with you. See below for more details about this year's Sponsors' Exhibition!

    - **Revel and End of Year Dinner ** - The two formal events hosted by DoCSoc, which will give you the opportunity to celebrate with your coursemates before revising for exams and at the end of the summer term, with a unique atmosphere which you will remember even after you have graduated!